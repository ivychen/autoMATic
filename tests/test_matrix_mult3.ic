int main() {
    float matrix x = [[1.0,2.0];[3.0,4.0]] * [[2.0,0.0];[1.0,2.0]];
    print(x[0][0]); // 4.0
    print(x[0][1]); // 4.0
    print(x[1][0]); // 10.0
    print(x[1][1]); // 8.0
    return 0;
}
