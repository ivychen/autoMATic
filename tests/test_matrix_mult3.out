4.000000
4.000000
10.000000
8.000000
